<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtisWindowClass</class>
 <widget class="QDialog" name="AtisWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AtisWindow</string>
  </property>
  <widget class="QLabel" name="window_title">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New/Edit Atis</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="airport">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Select Airport</string>
   </property>
  </widget>
  <widget class="QLabel" name="airport_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Airport :</string>
   </property>
  </widget>
  <widget class="QLabel" name="runway_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Runway :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="runway">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="frequency_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frequency">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="modulation_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Modulation :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="modulation">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>AM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FM</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="relay_unit_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Relay Unit :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="relay_unit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Select Relay Unit</string>
   </property>
  </widget>
  <widget class="QLabel" name="power_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Power :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="power">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
  <widget class="QListWidget" name="tower_frequencies">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tower_frequencies_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tower Frequencies : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_frequency_btn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rm_frequency_btn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="tacan">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>126</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="tacan_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tacan :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tacan_band">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Y</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="ils_fq_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ils Frequency : </string>
   </property>
  </widget>
  <widget class="QLabel" name="srs_path_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>500</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SRS Path :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="srs_path">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>C:\SRS</string>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="ils_rwy_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ils Runway : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ils_fq">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="ils_rwy">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
